src/SignalR/clients/ts/signalr/src/HubConnection.ts

```diff
@@ -544,7 +544,11 @@ export class HubConnection {
                             if (message.type === MessageType.Completion) {
                                 delete this._callbacks[message.invocationId];
                             }
-                            callback(message);
+                            try {
+                                callback(message);
+                            } catch (e) {
+                                this._logger.log(LogLevel.Error, `Stream callback threw error '${e}'.`);
```

Comment:
```suggestion
                                this._logger.log(LogLevel.Error, `Stream callback threw error '${e.stack}'.`);
```

I think we should probably audit the other places we log errors for this.